helpers/test-data-generator.js

```diff
@@ -18,15 +18,11 @@ import {
 import testHelper from "../helpers/test-helper";
 
 module.exports = {
-  async setupAssignedTask(
-    colonyNetwork,
-    colony,
-    dueDate = testHelper.currentBlockTime(),
```

Comment:
Can't we do the default `dueDate` setting here by just awaiting the `testHelper` call?

Reply:
Pretty sure I tried this, and you can't do it.